src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,207 @@
+package tasks;
+import exceptions.EkudIllegalArgException;
+import ui.Ui;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TaskList handles task modifications and invalid user inputs for
+ * modifying tasks, and is a key component of the Ekud chatbot.
+ */
+public class TaskList {
+    // Actual list storing the tasks
+    private List<Task> tasks;
+    // Constructor for TaskList
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Prints this TaskList to the console.
+     */
+    public void showTasks() {
+        if (this.tasks.isEmpty()) {
+            Ui.printMsg("Your to-do list is currently empty :o");
```

Comment:
Is it a good idea to allow the TaskList class to use static methods of the Ui class? In my opinion, the Ui class should be the one calling these methods. However, this works so all's good!